src/components/Signup.vue

```diff
@@ -82,13 +86,15 @@ export default {
       error: ''
     }
   },
+  computed: {
+    passwordMatch () {
+      return this.newUser.password === this.newUser.confirm
+    }
+  },
   methods: {
     register () {
-      if (this.newUser.password !== this.newUser.confirm) {
-        this.error = 'Your passwords do not match.'
+      if (!this.passwordMatch) {
```

Comment:
Modified. Should be cleaned up some more and shouldn't accept blank passwords now.